sardana_icepap/macro/esync.py

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env python
+
+from tango import DevState, DevFailed
+import time
+import json
+from sardana.macroserver.macro import Macro, Type
+
+
+def isIcepapMotor(macro, motor):
+    """
+    Checks if pool motor belongs to the IcepapController
+
+    Args:
+            macro (Macro): running macro
+            motor (Type.Motor): motor device
+    """
+    controllers = macro.getControllers()
+    ctrl_name = motor.controller
+    controller = controllers[ctrl_name]
+    return isIcepapController(controller)
+
+
+def isIcepapController(controller):
+    """
+    Checks if pool controller is of type IcepapController
+
+    Args:
+            controller (Type.Controller): controller device
+    """
+    controller_class_name = controller.getClassName()
+    return controller_class_name == "IcepapController"
+
+
```

Comment:
It is defined on utils.py

Reply:
actually is not being called in the macro any longer